tests/integration/test_arithmetic.py

```diff
@@ -112,7 +112,7 @@ def main(a1: angle, a2: angle) -> bool:
         a3 = -a1 + a2 * -3
         a3 -= a1
         a3 += 2 * a1
-        return a3 / 3 == -a2
+        return a3 == -a2
```

Comment:
Isn't the idea of this being a rotation rather than a true angle, that you *can* divide by 3? So we can keep the previous test?

Reply:
after all, we have the new one below too

Reply:
this isn't an execution test, though. Might be better if it were!

Reply:
Change reverted, have not made into an execution test or anything